gost-engine.h

```diff
@@ -0,0 +1,14 @@
+/**********************************************************************
+ *                            gost-engine.h                           *
+ *                     GOST engine in library form                    *
+ *                                                                    *
+ *      Copyright (c) 2021 Richard Levitte <richard@levitte.org>      *
```

Comment:
Uhmmmm, did I write this file?  :wink:

Reply:
Seems to be true)
https://github.com/gost-engine/engine/blob/62583fb222ec89ff4f6aa3d18b91ed3e64ed5cea/gost-engine.h

Reply:
Oh, I did!  Ok then...  but why are you adding it again?  It's already in place in master...

Reply:
It's needed in 1_1_1 branch. As I found in Readme that's the only branch to build against openssl-1.1.1.

Reply:
Oh!  I didn't notive that you were working against the 1.1.1 branch.  Sorry...